packages/cloudflare/src/api/cloudflare-context.ts

```diff
@@ -250,12 +253,13 @@ async function monkeyPatchVmModuleEdgeContext(cloudflareContext: CloudflareConte
 async function getCloudflareContextFromWrangler<
   CfProperties extends Record<string, unknown> = IncomingRequestCfProperties,
   Context = ExecutionContext,
->(): Promise<CloudflareContext<CfProperties, Context>> {
+>(options?: Omit<GetPlatformProxyOptions, "environment">): Promise<CloudflareContext<CfProperties, Context>> {
   // Note: we never want wrangler to be bundled in the Next.js app, that's why the import below looks like it does
   const { getPlatformProxy } = await import(/* webpackIgnore: true */ `${"__wrangler".replaceAll("_", "")}`);
   const { env, cf, ctx } = await getPlatformProxy({
     // This allows the selection of a wrangler environment while running in next dev mode
     environment: process.env.NEXT_DEV_WRANGLER_ENV,
+    ...options,
```

Comment:
Change the order (to protect against javascript or cast)
```suggestion
    ...options,
    environment: process.env.NEXT_DEV_WRANGLER_ENV,   
```